misc/semver-checker/modules/semver-checker-core/build.gradle

```diff
@@ -26,8 +27,13 @@ dependencies {
     implementation project(':ballerina-parser')
     implementation project(':central-client')
     implementation project(':ballerina-tools-api')
-    implementation('com.google.code.gson:gson')
     testCompile 'org.testng:testng'
+    //gson library for handling json object
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.9'
+    implementation group: 'org.json', name: 'json', version: '20211205'
+    implementation("com.google.guava:guava:31.0.1-jre")
+    compile group: 'com.google.code.gson', name: 'gson', version: '2.8.5'
+    compile group: 'com.google.guava', name: 'guava', version: '27.1-jre'
```

Comment:
Please remove all the duplicated and multiple versions of the guava library. Also whats the use case that we specifically need to use the guava library?